mock-identity-system/src/test/java/io/mosip/mock/identity/system/MockIdentitySystemApplicationTest.java

```diff
@@ -0,0 +1,8 @@
+package io.mosip.mock.identity.system;
+
+/**
+ * Unit test for simple App.
+ */
+public class MockIdentitySystemApplicationTest 
+{
```

Comment:
Can we also add a few integration test scenarios which will test end to end scenarios